packages/@atjson/offset-annotations/src/annotations/cne-audio-embed.ts

```diff
@@ -1,7 +1,8 @@
 import { BlockAnnotation } from "@atjson/document";
 
 export class CneAudioEmbed extends BlockAnnotation<{
-  url: string;
```

Comment:
@tim-evans With URL I wanted to use the same URL the user used in input. Thinking in having different environments:

- Sandbox, ie: https://embed-audio-sandbox.cnevids.com/script/episode/31d77731-e145-421c-a43b-4c9d60c91a00?skin=vf&target=js-audio1
- Production, ie: https://embed-audio.cnevids.com/script/episode/565452b5-fd3c-4329-8952-0a866ab20469?skin=vf

What do you think is the best option to proceed in this scenario?

Reply:
I'm not sure if it makes sense to provide this, no? This seems more like something that should be controllable by the hosting application rather than something that is controlled by edit.